index.html

```diff
@@ -489,49 +498,47 @@
-            whose bitmap data is provided by the <a>input media stream
-            track</a>.
-          </p>
-        </div>
+      <div>
         <p>
-          When a user agent is required to <dfn>fire a video monitor
-          event</dfn>, it must <a>fire a video worker event</a> named
-          <a>videomonitor</a>.
+          The <dfn><code>trackId</code></dfn> attribute must return the value
+          it was initialized to. When the object is created, this attribute
+          must be initialized to the empty string. It represents the identifier
+          that is shared with the <a>video monitor</a> and its <a>input media
+          stream track</a>.
         </p>
-      </section>
-      <section>
-        <h2>
-          <code>VideoProcessorEvent</code> interface
-        </h2>
         <p>
-          The <a>video processor event</a> inherits from the <a>video monitor
-          event</a>, and in addition provides means to programmatically
-          construct an <a>output frame</a> for the <a>output media stream
-          track</a>. It uses the <a>VideoProcessorEvent</a> interface for its
-          <a>videoprocess</a> events:
+          <!-- TODO: is this current stream position? -->
+           The <dfn><code>playbackTime</code></dfn> attribute must return the
+          value it was initialized to. When the object is created, this
+          attribute must be initialized to zero. It represents the current
+          stream position, in seconds.
         </p>
-        <pre class="idl">
-          [Constructor(DOMString type, optional VideoProcessorEventInit videoProcessorEventInitDict),
-           Exposed=Worker]
+        <p>
+          The <dfn><code>inputImageBitmap</code></dfn> attribute must return
+          the value it was initialized to. When the object is created, this
+          attribute must be initialized to null. It represents the
+          <a>ImageBitmap</a> object whose bitmap data is provided by the
+          <a>input media stream track</a>.
+        </p>
+      </div>
+      <p>
+        When a user agent is required to <dfn>fire a video monitor event</dfn>,
+        it must <a>fire a video event</a> named <a>videomonitor</a>.
+      </p>
+    </section>
+    <section>
+      <h2>
+        <code>VideoProcessorEvent</code> interface
+      </h2>
+      <p>
+        The <a>video processor event</a> inherits from the <a>video monitor
+        event</a>, and in addition provides means to programmatically construct
+        an <a>output frame</a> for the <a>output media stream track</a>. It
+        uses the <a>VideoProcessorEvent</a> interface for its
+        <a>videoprocessorchange</a> events:
+      </p>
+      <pre class="idl">
+          [Constructor(DOMString type, optional VideoProcessorEventInit videoProcessorEventInitDict)]
           interface VideoProcessorEvent : VideoMonitorEvent {
                           attribute ImageBitmap? outputImageBitmap;
```

Comment:
Just make a note that the `outputImageBitmap` attribute should have type `Promise<ImageBitmap?>`, see #38. Or, even replace this attribute with a setting method, see #39. Of course, this issue does not belong to the scope of this PR, we should handle it separately.